sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/SonarAnalyzer.CSharp.csproj

```diff
@@ -29,7 +33,17 @@
   </ItemGroup>
 
   <ItemGroup>
-    <EmbeddedResource Include="CBDE/windows/dotnet-symbolic-execution.exe" />
+    <EmbeddedResource Include="$(dotnet_symbolic_execution_windows)"  />
+    
+    <Content Include="$(dotnet_symbolic_execution_windows)" >
+      <Link>CBDE\windows\dotnet-symbolic-execution.exe</Link>
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </Content>
+    
+    <Content Include="$(cbde_dialect_checker_windows)" >
+      <Link>CBDE\windows\cbde-dialect-checker.exe</Link>
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </Content>

```

Comment:
These two content includes are needed only for unit tests (they replaced the logic from the `DownloadCBDE.targets` which was removed).